definitions/divorce/json/AuthorisationCaseState/AuthorisationCaseState-resp-journey-nonprod.json

```diff
@@ -0,0 +1,296 @@
+[
+  {
+    "LiveFrom": "01/02/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseStateID": "AosAwaiting",
```

Comment:
please see if this (state, role) authorisation doesn't exist yet. 

It does:
```
  {
    "LiveFrom": "26/03/2021",
    "CaseTypeID": "DIVORCE",
    "CaseStateID": "AosAwaiting",
    "UserRole": "caseworker-caa",
    "CRUD": "CRU"
  },
```

in: `definitions/divorce/json/AuthorisationCaseState/AuthorisationCaseState.json:3201`

Reply:
yes. all this file is redundant. 
removed it.